an-apluss/epic-mail
# [finish ft-signin-164297123] Add signin page

#### What does this PR do?
- This display a page to take user's input

#### Description of Task to be completed?
- Create a user interface to accept user input

#### How should this be manually tested?
- Ensure you have access to internet
- Ensure you have web browser installed
- Click [link]( https://an-apluss.github.io/epic-mail/UI/index.html )

#### What are the relevant pivotal tracker stories?
- PT Story link - ( `[164297123](https://www.pivotaltracker.com/story/show/164297123)`)

#### Screenshots 
![sign in - epic - an-apluss github io](https://user-images.githubusercontent.com/45961117/53899529-ae6a6580-403a-11e9-9e30-dd10babf9605.png)
